frontend/src/pages/CalendarPage/index.js

```diff
@@ -123,6 +123,18 @@ class CalendarPage extends React.Component {
     } = this.state;
     const { classes, history } = this.props;
 
+    // color of events based on status
```

Comment:
What do you think about having this within the Calendar component instead of being in the CalendarPage? I think there's merit in doing so because I don't see CalendarPage as being responsible for controlling *how* the Calendar component should color the events.

Specifically, resources is of an object type that's directly from the dx-scheduler component. Having to define resources outside of the Calendar component tightly couples our CalendarPage to our *implementation* of the Calendar component. For example, if we wanted to switch away from using dx-scheduler in the future, we'd have to change both the CalendarPage and the Calendar component - this means that the abstraction that the Calendar component provides is a "leaky" one.